composable.js

```diff
@@ -96,6 +96,13 @@
         innerText: function (node) {
             return node ? node.innerText || node.textContent : null;
         },
+        rootInnerText: function (node) {
+            var baseText = Composable.T.reverse(Composable.T.innerText(node));
```

Comment:
Is it possible to get all of the direct descendant text nodes and put their `innerText` together rather than getting `innerText` of the root and removing text of indirect descendants? It seems like it might be a more intuitive process.